test/spartan_SUITE.erl

```diff
@@ -90,6 +96,28 @@ generate_fixture_mesos_zone() ->
             data = #dns_rrdata_ns{
                 dname = <<"ns.spartan">>
             }
+        },
+        #dns_rr{
+            name = <<"_service._tcp.marathon.mesos">>,
```

Comment:
Lets change the test to have non mesos records. This could give wrong impression that HTTP API could be used to resolve mesos domain